lua/lvim/core/breadcrumbs.lua

```diff
@@ -0,0 +1,204 @@
+local M = {}
+
+-- local Log = require "lvim.core.log"
+
+M.config = function()
```

Comment:
this should be `M.setup`
and we might need a `M.config`

I'll address these in a PR